### DISCUSSION TITLE
mikrotik-routeros script additional storage_type

### DISCUSSION THREAD

**Discussion by mcrtpl** [tteck/Proxmox#2514] [Ideas]
Could you add additional storage_type to mikrotik-routeros script:
zfspool)
  DISK_EXT=""
  DISK_REF="$VMID/"
  DISK_FORMAT="subvol"
  DISK_IMPORT="-format raw"
  ;;

**Comment by tteck** [comment]
```
btrfs | zfspool)
  DISK_EXT=".raw"
  DISK_REF="$VMID/"
  DISK_FORMAT="subvol"
  DISK_IMPORT="-format raw"
```
https://github.com/tteck/Proxmox/commit/94cf9da7a1b23b5189b2b5a379b13a97bc8be94c

**Reply by mcrtpl** [reply]
zfspool doesnt have DISK_EXT=".raw"

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/commit/8d55efd704e321f38aeba19004396e234774b61d

**Reply by mcrtpl** [reply]
while the btrfs probably does, dunno not using btrfs

**Reply by tteck** [reply]
Did it work for you?

**Reply by mcrtpl** [reply]
with the zfspool yes
